CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+Fix: update type and name of device (#770)
```

Comment:
Maybe a better description whould be this one?

```suggestion
Fix: PUT /iot/devices/:deviceId operation to implement proper update of entity_name and entity_type (#770)
```

Please, check if I'm right before accepting it (I may be wrong)

Reply:
Fixed in b7817e0